install_config/persistent_storage/topics/glusterfs_uninstall.adoc

```diff
@@ -0,0 +1,22 @@
+For {gluster-native}, the {product-title} Advanced Installer comes with a
```

Comment:
I'm not sure what version this PR is intended for, but I think for 3.10 we don't use "advanced install" anymore, because the "quick install" won't exist in the 3.10 docs (I could be wrong). It might be better to cover that and just say "For {gluster-native}, an {product-title} install comes with...".